docs/bql.md

```diff
@@ -357,10 +357,20 @@ keywords. They can be composed together using ```not```, ```and```, and
   }
   AFTER 2006-01-01T15:04:05.999999999Z07:00
 ```
-
+If you have multiple predicates the After/Before Syntax has to either specify
+the predicate of interest or that it should be applied to all predicates in 
+the following manner: 
+```
+  SELECT ?user
+  FROM ?social_graph
+  WHERE {
+    ?user "folows"@[,] /user<Joe> .
+    ?user "interacts"@[,] /user<Joe>
+  }
+  AFTER ""@[2006-01-01T15:04:05.999999999Z07:00]
```

Comment:
In grammar_test there is an example of valid After syntax and it seems to be the one I mention, as seen in:https://github.com/google/badwolf/blob/579facbd49bd5a0b5f02e4fb13056c31c30ad6c2/bql/grammar/grammar_test.go#L82

Do you think which one should be the correct one?

Reply:
Apologies for the slow answer @rbkloss. Yes, than sounds like an oversight since it is not used in the current implementation. I think we should fix it.

Reply:
@rbkloss, actually @aerostitch just sent a pull request to fix the code to align the cod with the documentation. See PR #101. You may want to revisit this write up after that since the documentation should be mostly correct after it gets merged.

Reply:
He linked his PR to this one so if that gets approved this one gets closed, with that said, I think we are all setup, WDYT?